src/main/java/anya/Ui.java

```diff
@@ -0,0 +1,111 @@
+package anya;
+
+import java.util.Scanner;
+
+import anya.task.Task;
+import anya.task.TaskList;
+
+public class Ui {
+
+    private Scanner sc;
+
+    // Constructor
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    // Instance methods
+    private void breakLine() {
+        System.out.println("---------------------------------------------------------------------");
+    }
+
+    public String readLine() {
+        return this.sc.nextLine();
+    }
+
+    public void closeScanner() {
+        this.sc.close();
+    }
+
+    public void greetMessage() {
+        System.out.println("Hello! Anya is happy to meet you.\nHow can Anya help?");
+        breakLine();
+    }
+
+    public void exitMessage() {
+        System.out.println("Anya is sad to see you leave. Please be back soon.");
+        breakLine();
+    }
+
+    public void addTaskMessage(Task task, int taskListLength) {
+        System.out.println("Anya added: " + task);
+        System.out.println("Anya sees that you have " + taskListLength + " task(s) in your list.");
+        breakLine();
+    }
+
+    public void deleteTaskMessage(Task task) {
```

Comment:
Perhaps a clearer method name here? E.g. `showDeleteTaskMessage`. I noticed the same issue in several other methods too.